### DISCUSSION TITLE
Duplicate Groups within Pods/

### DISCUSSION THREAD

**Discussion by itxplor** [pods-framework/pods#7322] [Ideas]
You can currently duplicate entire pods, or fields but not field groups within pods.  It would be super useful to be able to duplicate entire groups of pods, even if its just within that same pod.  Thank you!

**Comment by sc0ttkclark** [comment]
Heck yes that's a great idea!

**Reply by silentdial** [reply]
Read my mind, I'm trying Pods right now with Local by FlyWheel, spent a couple hours trying to make a repeatable group (real estate listings website). This would be HUGELY useful feature for us. I'd happily buy an annual license if this could work.

**Comment by sc0ttkclark** [comment]
Resolved via https://github.com/pods-framework/pods/pull/7426
